frontend2/src/contexts/EpisodeContext.ts

```diff
@@ -0,0 +1,10 @@
+import { createContext } from "react";
+import { DEFAULT_EPISODE } from "../utils/constants";
+
+export const EpisodeContext = createContext({
+  // the default episode.
+  episode: DEFAULT_EPISODE,
+  setEpisode: (episode: string) => {
```

Comment:
Q: should we standardize how we refer to the current episode? I have been calling it `episodeId` (referring to `name_short` like `bc23`) but I would be OK with using `episode` instead.

Reply:
We should def standardize this

What about calling name_short `episodeId` and calling name_long `episodeName`?

Reply:
I agree with this ^^